src/Microsoft.ML.Data/Transforms/TermTransformer.cs

```diff
@@ -42,7 +42,7 @@ namespace Microsoft.ML.Transforms.Categorical
     // * The Key value is the one-based index of the item in the dictionary.
     // * Not found is assigned the value zero.
     /// <include file='doc.xml' path='doc/members/member[@name="TextToKey"]/*' />
-    public sealed partial class TermTransform : OneToOneTransformerBase
+    public sealed partial class TermTransformer : OneToOneTransformerBase
```

Comment:
>TermTransformer [](start = 32, length = 15)

Should we keep TermTransformer or use ValueToKeyMappingTransformer? To make it ActionPerformingTransformer